lib/internal/errors.js

```diff
@@ -1104,18 +1104,28 @@ E('ERR_INVALID_PACKAGE_CONFIG', (path, message, hasMessage = true) => {
 }, Error);
 E('ERR_INVALID_PACKAGE_TARGET',
   (pkgPath, key, subpath, target, base = undefined) => {
+    const relError = typeof target === 'string'
+      && target.length && !target.startsWith('./');
     if (key === null) {
       if (subpath !== '') {
         return `Invalid "exports" target ${JSONStringify(target)} defined ` +
         `for '${subpath}' in the package config ${pkgPath} imported from ` +
-        base;
+        `${base}.${relError ? ' - targets must start with "./"' : ''}`;
       } else {
         return `Invalid "exports" main target ${target} defined in the ` +
-          `package config ${pkgPath} imported from ${base}.`;
+          `package config ${pkgPath} imported from ${base}.${relError ?
+          ' - targets must start with "./"' : ''}`;
       }
     } else if (key === '.') {
       return `Invalid "exports" main target ${JSONStringify(target)} defined ` +
-      `in the package config ${pkgPath}${sep}package.json`;
+      `in the package config ${pkgPath}${sep}package.json${relError ?
+            ' - targets must start with "./"' : ''}`;
+    } else if (typeof target === 'string' && target !== '' &&
+      !target.startsWith('./')) {
+        return `Invalid "exports" target ${JSONStringify(target)} defined for '${
+          StringPrototypeSlice(key, 0, -subpath.length || key.length)}' in the ` +
+          `package config ${pkgPath}${sep}package.json. ` +
+          '- targets must start with `./`';
```

Comment:
Did you intend for this to be on a new line?

Reply:
probably not. I moved things around due to the linter.

Reply:
Well, I meant the message printed since it doesn't seem like proper grammar, but may be fine as long as it remains consistent with the other errors that include a dash like this:

https://github.com/nodejs/node/blob/91e8e80c9261982a140752e6ba821f80b5451a9c/lib/internal/errors.js#L1117-L1118